src/components/GiftCard/Checkout.tsx

```
@@ -76,9 +93,9 @@ const Checkout = ({ data, openCheckoutModal, handleCloseCheckoutModal, refetchCa
             </label>
             <select value={paymentMethod} onChange={handleChangePaymentMethod} id="paymentMethod" className="border px-3 py-2" required>
               <option value="">------</option>
-              <option value="cash">Cash</option>
-              <option value="EDC BCA">EDC BCA</option>
-              <option value="EDC Mandiri">EDC Mandiri</option>
+              {methods.map((method: any) => {
+                return <option value={method.name}>{method.name}</option>;
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)